rskj-core/src/main/java/co/rsk/core/RskFactory.java

```diff
@@ -415,7 +415,13 @@ public TransactionGateway getTransactionGateway(
     public BuildInfo getBuildInfo(ResourceLoader resourceLoader) throws IOException {
         Properties props = new Properties();
         Resource buldInfoFile = resourceLoader.getResource("classpath:build-info.properties");
-        props.load(buldInfoFile.getInputStream());
+        try {
```

Comment:
This is the 🥔, great work with the previous refactoring. Please remove the now unnecessary exception!